.ci/bindist/linux/debian/focal/buildinfo.json

```diff
@@ -1,5 +1,5 @@
 {
-  "docker": "docker.pkg.github.com/clash-lang/clash-compiler/bindist-debian-focal:2021-05-14",
+  "docker": "docker.pkg.github.com/clash-lang/clash-compiler/bindist-debian-focal:2021-12-08",
```

Comment:
You probably need to redo this after https://github.com/clash-lang/clash-compiler/pull/2043..